metrics/network/nuttcp_dockerfile/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Usage: FROM [image name]
-FROM ubuntu
+FROM ubuntu:16.04
```

Comment:
why 16.04? that is an old image from ubuntu

Reply:
It seems huttcp image is never used by metrics (I grep'ed the logs of the metrics jobs and found nothing). So I was not sure how to test it with ubuntu 20.04. Because that files was created at the same time of [metrics/network/iperf_dockerfile/Dockerfile](https://github.com/kata-containers/tests/tree/main/metrics/network/iperf_dockerfile/Dockerfile) (which uses ubuntu 16.04 too) I thought 16.04 would be the safest value.

What version should I use instead? Or is it safe to keep using `latest`?

Reply:
I changed it to use latest instead of 16.04.